tests/lib/ansible_helper.py

```diff
@@ -145,8 +146,18 @@ def create_inventory(self, nodes, working_dir=None):
         with open(inv_path, 'w') as inv_file:
             yaml.dump(inv, inv_file)
 
+        logger.info('Inventory path: {}'.format(inv_path))
         return inv_path
 
+    def run_play_raw(self, playbook):
+        path = os.path.join('tests/assets/ansible', playbook)
+        logger.info(f'Running playbook {path}')
+        try:
+            subprocess.run(['ansible-playbook', '-i', self.inventory_file,
+                            path])
+        except subprocess.CalledProcessError:
+            logger.error('An error occured executing Ansible playbook')
```

Comment:
1) please use `logger.exception()` to also have the traceback in the logs
2) I think we should reraise the exception to stop here in case of an error.